mantidproject/mantid
# IndirectILLEnergyTransfer: fix the sign in the formula

This PR fixes the sign of the energy transfer formula in IndirectILLEnergyTransfer.
(system test reference has been updated).

**To test:**
1. Set ILL as your facility. Untar the attachment. Run: 
```
IndirectILLReductionQENS(Run='225328-225356', 
                                              SumRuns=True, 
                                              CropDeadMonitorChannels=True, 
                                              OutputWorkspace='out', 
                                              SpectrumAxis='2Theta')
GroupDetectors(InputWorkspace='out_2theta', WorkspaceIndexList='0-17', OutputWorkspace='grouped')
```
2. Plot the grouped spectrum.
3. Zoom in the region X: [-0.01, 0.01] and Y: [0,400]
4. Confirm that the right shoulder of the peak is slightly higher than the left shoulder (by eye should be enough).
 
[data4PR.zip](https://github.com/mantidproject/mantid/files/2053100/data4PR.zip)


Fixes #22493. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
